cpp/data/sql_updates/ub_tools.24

```diff
@@ -0,0 +1,7 @@
+DELETE * FROM metadata_presence_tracer;
```

Comment:
I guess you should add `WHERE 1` add the end, because some MySQL installations my refuse to execute a `DELETE` statement without `WHERE` (especially under CentOS).